harness-delegate-ng/templates/deployment.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - containerPort: {{ .Values.service.port }}
+            - name: api
+              containerPort: 3460
```

Comment:
thanks, it's in general good practice to name and list the ports in container spec anyways.
Just a nit, would it be better to name it something more distinguishable. "api" is a general term. But up to you.

Reply:
@xingchi-jin  I dont know too much about the service running on this port, if there is a more descriptive name than `api` then I would use that. All i use it for is getting delegate metrics to prometheus->grafana. The endpoint is `/api/metrics` thus `api`